app/models/ConfText.scala

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2019, German Neuroinformatics Node (G-Node)
+//
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted under the terms of the BSD License. See
+// LICENSE file in the root of the Project.
+
+package models
+
+import javax.persistence.{Entity, ManyToOne}
+import models.Model.{unwrapRef}
+
+/**
+ * Model for conference texts
+ */
+@Entity
+class ConfText extends Model {
+
+  var ctType: String = _
+  var text: String = _
+
+  @ManyToOne
+  var conference: Conference = _
+
+}
+
+
+object ConfText {
+
+  def apply(uuid: Option[String],
+            ctType: Option[String],
+            text: Option[String],
+            conf: Option[Conference] = None) : ConfText = {
+
+    val confText = new ConfText()
+
+    confText.uuid       = unwrapRef(uuid)
+    confText.ctType     = unwrapRef(ctType)
+    confText.text       = unwrapRef(text)
```

Comment:
This needs an `@Column(length=512)` annotation, otherwise the import of `description` will fail.

As an additional note: the front-end in admin-conference needs to make sure that both `description` and `notice` are longer than 512 characters, otherwise an uncaught exception will occur when trying to save.